drivers/timer/stm32_lptim_timer.c

```diff
@@ -61,11 +64,63 @@ static const struct device *const clk_ctrl = DEVICE_DT_GET(STM32_CLOCK_CONTROL_N
  *    0xFFFF / (LSE freq (32768Hz) / 128)
  */
 
-static uint32_t lptim_clock_freq = KHZ(32);
-static int32_t lptim_time_base;
+#if (CONFIG_STM32_LPTIM_CLOCK_LSI)
+/*
+ * Most of the STM32_SERIES have a LSI @32000Hz,
+ * this frequency might differ on some that do not yet use LPTIM
```

Comment:
LPTIM is available on L0.
There is some dedicated code around it. See https://github.com/zephyrproject-rtos/zephyr/blame/main/drivers/timer/stm32_lptim_timer.c#L368

Reply:
I guess this comment is not relevant at this place anymore.